src/pyshark/tshark/tshark_xml.py

```diff
@@ -5,6 +5,10 @@
 from pyshark.packet.packet import Packet
 from pyshark.packet.packet_summary import PacketSummary
 
+# Prepare dictionary used with str.translate for removing invalid XML characters
+DEL_BAD_XML_CHARS = { c: None for c in range(0x00, 0x20) if not c in (0x09, 0x0a, 0x0d) }
```

Comment:
Please remove the spaces in the dict beginning `{ ` use a longer variable name than `c`.

Reply:
Updated. Thank you for checking and for considering these changes.